### DISCUSSION TITLE
application template

### DISCUSSION THREAD

**Discussion by srvarey** [primefaces/community#1670] [PrimeVue] [answered]
I bought the Poseidon application template (from here https://primevue.org/templates/)
All the pages in the application only take up 50% of the screen . I have a 32 in monitor and a huge waste of screen real estate - especially when i want to view large graphs.
There is no documenation . Does anyone know if the page width is configurable? or how i can increase page width application scope?
thanks in advance.

**Comment by tugcekucukoglu** [comment, marked as answer]
You could change the styles from `responsive.scss`.

**Reply by srvarey** [reply]
No such file exists. If I create it , what should it look like?
